### ISSUE TITLE
ASW performance question

### ISSUE THREAD

**Issue by DevKiHyun** [Jiang-Yidi/TS-TalkNet#3]
Hi,

I could run some experiments for  ASW dataset by your help!

Actually, we also tried to reproduce just TalkNet-baseline and TS-talkNet performances of yours.

I wonder that how many epoch you run for TalkNet-baseline with ASW train dataset?

You also not calculate  the loss of frames which have 0 eval label in training step?

Thanks!

**Comment by Jiang-Yidi** [comment]
'I wonder that how many epoch you run for TalkNet-baseline with ASW train dataset?'
--> 25 epoch

'You also not calculate the loss of frames which have 0 eval label in training step?'
--> I didn't exclude the 0 eval labels in training phase and only consider them in evaluation.

**Comment by DevKiHyun** [comment]
Thanks!
